prow/plugins/trigger/trigger.go

```diff
@@ -81,6 +81,13 @@ func helpProvider(config *plugins.Configuration, enabledRepos []config.OrgRepo)
 		WhoCanUse:   "Anyone can trigger this command on a trusted PR.",
 		Examples:    []string{"/retest"},
 	})
+	pluginHelp.AddCommand(pluginhelp.Command{
```

Comment:
I think we should also extend the description of `test` and mention that it will list all possible jobs when no jobs/an invalid job was specificed. Also change the `Usage` to `"/test [<job name>|all]"` to clarify that the target argument became optional.

Reply:
Thank you for the review. Fixed.

Reply:
Does the PR need to be trusted for this command to work?

Reply:
Thank you for the review. Currently yes. It seemed to me that it could be confusing if available presubmits were listed in response to a help comment, when the user (who is not trusted) cannot run any of them.

Reply:
@cblecker WDYT?

Reply:
Seems reasonable to me